Tests/SentryTests/SentryBinaryImageCacheTests.swift

```diff
@@ -226,6 +226,38 @@ class SentryBinaryImageCacheTests: XCTestCase {
         
         waitForExpectations(timeout: 1)
     }
+    
+    func testStartWithSenryCrashBinaryImageCache() {
```

Comment:
`l`: would be great if the title of the test or comments would give more details on what exactly  is tested here